### PR TITLE
CORE-780: Fix cryptoNetworkInstallBuiltins Handling Non-Native Units

### PR DESCRIPTION
**Pull request by EBGToo**

### CODE REVIEW

File: crypto/BRCryptoNetwork.c [hunk 1]
```diff
@@ -775,6 +773,9 @@ cryptoNetworkInstallBuiltins (BRCryptoCount *networksCount) {
                                                  currencySpec->type,
                                                  currencySpec->address);
 
+                BRCryptoUnit unitBase    = NULL;
+                BRCryptoUnit unitDefault = NULL;
```
Comment (michael-brd) [inline crypto/BRCryptoNetwork.c]: Is this leaked?

File: crypto/BRCryptoNetwork.c [hunk 2]
```diff
@@ -814,15 +815,16 @@ cryptoNetworkInstallBuiltins (BRCryptoCount *networksCount) {
         }
 
         // Create the Network Fees
+        BRCryptoUnit feeUnit = cryptoNetworkGetUnitAsBase (network, cryptoNetworkGetCurrency(network));
```
Comment (michael-brd) [inline crypto/BRCryptoNetwork.c]: Is feeUnit leaked? Is `cryptoNetworkGetCurrency()` result leaked?

**Comment by EBGToo** [comment]
I think I got them all handled.

File: crypto/BRCryptoNetwork.c (hunk 2 above)
Comment (michael-brd) [reply crypto/BRCryptoNetwork.c]: Isn't the cryptoNetworkGetCurrency result leaked?

Comment (EBGToo) [reply crypto/BRCryptoNetwork.c]: Fixed that.  And another once over.

Comment (michael-brd) [reply crypto/BRCryptoNetwork.c]: The joys of memory management; looks good!

**Pull request merged**